### PR TITLE
Script to run action locally

### PR DESCRIPTION
**Pull request by IgorDobryn**

### CODE REVIEW

**Review by jweslley — APPROVED** [review]

**Review by fuksito — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -4,5 +4,13 @@
     "fetch-task-definition",
     "read-from-ssm-param-store",
     "run-ecs-task"
-  ]
+  ],
+  "devDependencies": {
+    "commander": "^6.2.0",
+    "fs": "^0.0.1-security",
```
Comment (le0pard) [inline package.json]: WAT? node.js have build in `fs` module.

**Review by le0pard — CHANGES_REQUESTED** [review]
https://www.npmjs.com/package/fs - why?

File: package.json (hunk 1 above)
Comment (IgorDobryn) [reply package.json]: Good catch!

Comment (IgorDobryn) [reply package.json]: @le0pard updated.

**Review by le0pard — APPROVED** [review]

**Pull request merged**